src/App.js

```diff
@@ -1,25 +1,33 @@
 import './App.scss';
 import { Route, Routes } from 'react-router-dom';
-import  Navigation  from './components/Navigation/Navigation';
+import Navigation from './components/Navigation/Navigation';
 import Forms from './components/Forms/Forms';
-import ListRestorans from './components/ListRestorans/ListRestorans';
-import Footer from './components/Footer/Footer';
-import RestoranTopFood from './components/Restorans/RestoranTopFood';
+import HomePage from './pages/HomePage/HomePage';
+import FooterForRestoran from './components/Footer/FooterForRestoran';
+import RestoranTopFood from './pages/Restorans/RestoranTopFood';
+import Menu from './pages/Restorans/Menu/Menu';
+import Impressions from './pages/Restorans/Impressions/Impressions';
+import Gallery from './pages/Restorans/Gallery/Gallery';
+import FrequentlyAskedQuestions from './pages/AdditionalInformation/FrequentlyAskedQuestions';
+import PrivacyNotices from './pages/AdditionalInformation/PrivacyNotices';
+import TermsOfUse from './pages/AdditionalInformation/TermsOfUse';
 
 const App = () => {
   return (
     <div className='App'>
-       <Navigation /> 
-      <Forms />
-      <ListRestorans />
-      <Footer /> 
-      <RestoranTopFood/>
-      {/* <Routes>
-        <Route path='/navigation' element={<Navigation/>}></Route>
-        <Route path='/forms' element={<Forms/>} />
-        <Route path='/listrestoran' element={<ListRestorans/>} />
-        <Route path='/footer' element={<Footer/>} />
-      </Routes>  */}
+      <Routes>
+        <Route path='/topfood' element={<RestoranTopFood />} />
+        <Route path='/navigation' element={<Navigation />} />
+        <Route path='/forms' element={<Forms />} />
+        <Route path='/' element={<HomePage />} />
+        <Route path='/footer' element={<FooterForRestoran />} />
+        <Route path='/menu' element={<Menu />} />
+        <Route path='/impressions' element={<Impressions />} />
+        <Route path='/gallery' element={<Gallery />} />
+        <Route path='/askedquestions' element={<FrequentlyAskedQuestions />} />
```

Comment:
You should change `askedquestions` to `faq`. It is usual label for frequently asked questions, and your component is also called like that

Reply:
Change askedquestions to faq